LICENSE.txt

```diff
@@ -0,0 +1,621 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
```

Comment:
Is this the copyright notice for the license or the licensed code?

Reply:
For the license itself.